packages/esm-patient-chart-app/src/patient-chart/patient-chart.component.tsx

```diff
@@ -17,7 +17,7 @@ import Loader from '../loader/loader.component';
 import styles from './patient-chart.scss';
 import VisitHeader from '../visit-header/visit-header.component';
 import SideMenuPanel from '../side-nav/side-menu.component';
-import { getPatientChartStore } from '@openmrs/esm-patient-common-lib';
+import { getPatientChartStore } from '@openmrs/esm-patient-common-lib/src/store/patient-chart-store';
```

Comment:
We shouldn't need this change, right, @ibacher? `getPatientChartStore` should be resolved from automatically from the public API of `@openmrs/esm-patient-common-lib` without needing to import it deeply like so.

Reply:
Yes. In fact, deep imports like this have the potential to break the whole shared dependencies structure.

Reply:
(The only place we do really do things like this is with esm-framework, and there's a lot of configuration around ensuring that works).